gallery/src/styles/NxList/NxListBulletedExample.tsx

```diff
@@ -11,15 +11,40 @@ const NxListWithBullets = () =>
     <h4 className="nx-list__title">
       List title
     </h4>
-    <ul>
+    <ul className="nx-list nx-list--bulleted">
```

Comment:
So in this example the `.nx-list` styles are on _both_ the `div` and the top `ul`?  That doesn't seem right

Reply:
> So in this example the `.nx-list` styles are on _both_ the `div` and the top `ul`? That doesn't seem right

https://github.com/sonatype/sonatype-react-shared-components/pull/18/commits/08dfe7f04fdd3ffabf59b3d45d6adab914c0dc84